index.ts

```diff
@@ -18,6 +18,27 @@ type Issue = {
     pull_request: null | {},
     state: 'open' | 'closed',
     title: string,
+    body?: string;
+    assignee: {
+      login: string;
+      id: number;
+      node_id: string;
+      avatar_url: string;
+      gravatar_id: string;
```

Comment:
pls remove unnecessary fields which are not used in the code

Reply:
Fixed